src/App.tsx

```diff
@@ -0,0 +1,76 @@
+import { AppBar, Toolbar, Typography } from "@material-ui/core";
+import { Component, ReactNode } from "react";
+import { connect } from "react-redux";
+import "./App.css";
+import { Status } from "./common/constants";
```

Comment:
Configure the project to use absolute path imports. When the project becomes bigger they are much easier to work with.

Reply:
done